lib/router/utils.ts

```diff
@@ -12,6 +12,13 @@ export function isPromise<T>(p: any): p is Promise<T> {
   return p !== null && typeof p === 'object' && typeof p.then === 'function';
 }
 
+/**
+ * @private
+ */
+export function isFrozenOrNotExtensible(obj: object): boolean {
+  return Object.isFrozen(obj) || !Object.isExtensible(obj);
```

Comment:
When is `!Object.isExtensible(obj)` not enough? Specifically, I'm wondering if `!Object.extensible(obj)` can ever be true for frozen objects?

Reply:
You are right. I changed it to just `!Object.isExtensible(obj)` and ran on same browsers (with Babel polyfill included) and tests pass.

I'll try to see if I can get Travis happy as well if not then I guess have to merge it as is?